corehq/apps/userreports/views.py

```diff
@@ -1536,6 +1538,32 @@ def get_table(q):
         return export_response(Temp(path), params.format, adapter.display_name)
 
 
+@api_auth()
+@require_permission(HqPermissions.view_reports)
+@api_throttle
+def subscribe_to_data_source_changes(request, config_id):
```

Comment:
what is this change?

Reply:
Should we put this in a separate PR? It seems out of scope of what this PR was addressing and seems to have needed quite a few commits.

Reply:
Sorry for the confusion. Yes, that chunk of work was in [a separate PR](https://github.com/dimagi/commcare-hq/pull/33795), which has been merged into this branch.